README.md

```diff
@@ -10,4 +10,4 @@ Describe how you approached to problem, and what tools and techniques you used t
 
```

Comment:
Replace meee, please 🙏 

The recruiters are reading it sometimes. You can have the most rock&roll code in the repository... but without the nice cover, you will not be able to sell it!